src/pages/Script/types.ts

```diff
@@ -0,0 +1,73 @@
+export interface ScriptInfo {
+  scriptName: string
+  scriptType: string
+  typeId: number
+  codeHash: string
+  hashType: string
+  capacityOfDeployedCells: string
+  capacityOfReferringCells: string
+  countOfTransactions: number
+  countOfDeployedCells: number
+  countOfReferringCells: number
+  pageOfTransactions: number
+  pageOfDeployedCells: number
+  pageOfReferringCells: number
+  sizeOfTransactions: number
+  sizeOfDeployedCells: number
+  sizeOfReferringCells: number
+}
+
+export type ScriptTabType = 'transactions' | 'deployed_cells' | 'referring_cells' | undefined
+
+export interface ScriptGeneralInfoResponse {
+  id: number
+  codeHash: string
+  hashType: 'lock' | 'type'
+  scriptType: 'lock' | 'type'
+  typeId: number
```

Comment:
`type id` is a string, it's the `args` in executing type id contract

Ref: https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0022-transaction-structure/0022-transaction-structure.md#type-id